cmd/internal/switcher/sonic/redis/db/configdb.go

```diff
@@ -40,6 +40,15 @@ func newConfigDB(addr string, id int, separator string) *ConfigDB {
 	}
 }
 
+func (c *ConfigDB) IsInitialized(ctx context.Context) (bool, error) {
+	key := "CONFIG_DB_INITIALIZED"
+	result, err := c.rdb.Get(ctx, key).Result()
```

Comment:
`Get` will return an error of type redis.Nil if the key doesn't exist. We should use `Exists`.